core/trino-main/src/test/java/io/trino/sql/analyzer/TestAnalyzer.java

```diff
@@ -1335,45 +1337,123 @@ public void testWindowFrameTypeGroups()
     @Test
     public void testWindowFrameWithPatternRecognition()
     {
-        // in-line window specification
-        assertFails("SELECT rank() OVER (" +
+        analyze("SELECT rank() OVER (" +
                 "                           PARTITION BY x " +
                 "                           ORDER BY y " +
-                "                           MEASURES " +
-                "                               MATCH_NUMBER() AS match_no, " +
-                "                               LAST(A.z) AS last_z " +
+                "                           MEASURES A.z AS last_z " +
```

Comment:
Why did this change? Should we add a separate test case, instead?

Reply:
This test case was supposed to evolve from `NOT_SUPPORTED` to supported. But it was also incorrect, as `MATCH_NUMBER` is not legal in window.
I will fix the commit which introduces this case to make the query correct.